### ISSUE TITLE
WNDA3100v3 - Wireless Won't Connect

### ISSUE THREAD

**Issue by jadenPete** [jurobystricky/Netgear-A6210#12]
First, let me tell you, THANK YOU! I am SO very glad that someone is working on a driver for this device.

Here's what I did:
    1. Downloaded the archive from github
    2. Extracted the archive
    3. Opened a terminal and went into the directory
    4. Ran the `make` command
    5. Ran `sudo make install`
    6. Ran `modprobe mt7662u_sta` (the driver it created)

Then, I looked, and poof! The network manager started receiving the signals for the wifi networks and displaying the networks. I tried connecting to my network, but it failed. I installed wicd. And after a whole minute of trying to authenticate, it said I had a bad password. I was using WPA2 - the encryption I normally use with that router. I tried the password multiple times, and looked at it (showing it in plain text), and it was my password!

I am running Ubuntu 14.04.4 x86_64 (64bit, amd64, whatever).
I am running the Linux Kernel 4.2.0.34.27 (Wily) from the main repositories.
I am using a Netgear WNDA3100v3 (as displayed in the title).

Hope I can get some help with this, and thanks!

**Comment by jonipalosaari** [comment]
Google wpa_supplicant and try to configure it or use wep-encryption. Sounds like your driver works so this is not driver related issue.

**Comment by jadenPete** [comment]
Tried the tutorial on the forums, didn't do anything. So I tried the WEP encryption, and same thing happened. What's really weird, is that my router's logs don't report any attempt to connect to the network, but when attempting to connect, I get some network activity, as shown in the attachment.
![screenshot from 2016-03-23 13 49 31](https://cloud.githubusercontent.com/assets/13180372/14000357/285a8c2e-f0fe-11e5-8050-bbc84940d3bf.png)
![screenshot from 2016-03-23 13 49 52](https://cloud.githubusercontent.com/assets/13180372/14000358/28ebba96-f0fe-11e5-884a-b58806418796.png)
